spec.md

```diff
@@ -927,7 +927,7 @@ The following HTTP Headers are defined for this operation:
 
 ```
 {
-  "service_id": "service-id-here",
+  "service_id": "offering-id-here",
```

Comment:
`plan-id-here` seems ok because we don't say "Service Plan", just "Plan".

Reply:
I think it should actually be `service-plan-id`, as well as `service-offering-id` (see my note above about the definition of **Service Plan** instead of **Plan**)